tasks/system_probe.py

```diff
@@ -605,6 +606,57 @@ def ninja_generate(
         ninja_cgo_type_files(nw)
 
 
+def build_libpcap_static(ctx, dev_dir: str):
+    release = "1.10.5"
```

Comment:
why this release in particular? please add a comment

Reply:
There's no reason to use this version in particular, it just happens to be last release of libpcap as of now. I added this as a comment in cf23403.